Package.swift

```diff
@@ -27,15 +27,15 @@ let package = Package(
         .library(name: "SpeziSchedulerUI", targets: ["SpeziSchedulerUI"])
     ],
     dependencies: [
-        .package(url: "https://github.com/StanfordSpezi/SpeziFoundation", from: "2.1.4"),
-        .package(url: "https://github.com/StanfordSpezi/Spezi", from: "1.8.0"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziViews", from: "1.10.0"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziStorage", from: "2.1.0"),
+        .package(url: "https://github.com/StanfordSpezi/SpeziFoundation.git", from: "2.1.7"),
+        .package(url: "https://github.com/StanfordSpezi/Spezi.git", from: "1.8.2"),
+        .package(url: "https://github.com/StanfordSpezi/SpeziViews.git", from: "1.10.0"),
+        .package(url: "https://github.com/StanfordSpezi/SpeziStorage.git", from: "2.1.1"),
         .package(url: "https://github.com/StanfordSpezi/SpeziNotifications.git", from: "1.0.6"),
         .package(url: "https://github.com/apple/swift-algorithms.git", from: "1.2.0"),
-        .package(url: "https://github.com/swiftlang/swift-syntax", from: "601.0.0"),
+        .package(url: "https://github.com/swiftlang/swift-syntax.git", from: "601.0.0"),
         .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", from: "1.17.2"),
-        .package(url: "https://github.com/StanfordBDHG/XCTRuntimeAssertions.git", from: "1.1.3")
+        .package(url: "https://github.com/StanfordBDHG/XCTRuntimeAssertions.git", branch: "lukas/negative-assertions")
```

Comment:
TODO: update to tagged release once https://github.com/StanfordBDHG/XCTRuntimeAssertions/pull/21 is merged